### DISCUSSION TITLE
Is it possible to set an equivalent to --color=always?

### DISCUSSION THREAD

**Discussion by IgnacioJPickering** [Textualize/rich#2459] [Q&A]
I would like to print escape sequences even when writing to files, or to things that are not terminals for some use cases. I'm not sure if this is a feature that exists at the library level, or even if this is a responsibility of the library or is something that has to be set at the interpreter level or something else. In any case such a feature would be super useful.

Is this doable?

**Comment by willmcgugan** [comment]
`Console(force_color=True)`
